{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -26,30 +26,26 @@
 
 
 {% if cookiecutter.use_pytest == 'y' -%}
-class Test{{ cookiecutter.project_slug|title }}(object):
+@pytest.fixture
+def some_fixture():
+    pass
```

Comment:
Right, much better to discuss these things after having some code. :)

So, to be honest I don't really like the fixture from the example.
Although it's illustrative of what you can do with fixtures, it doesn't represent well what one would want to do with a test.

The sample code is basically testing the fixture, and for a real test one would want to have just setup code in the fixture, and the code to be exercised in the tests itself.

Also, I'm feeling the lack of a teardown-equivalent example in pytest -- it would be nice to mirror what we have in the unittest code.

I propose:

```
@pytest.fixture
def some_fixture():
    # put your setup code to run before the test here
    yield
    # put your teardown code to run after the test run
```

Btw, I'm not completely sure if the above is the equivalent of setup / teardown -- do you happen to know if the code after the yield would run even if the test code raised an exception?